girlpower/06_Loading_Geometry_Overview.v4p

```diff
@@ -18,7 +18,7 @@
    <PIN pinname="Size" slicecount="1" values="15">
    </PIN>
    </NODE>
-   <PACK Name="addonpack" Version="33.7.0">
```

Comment:
Is this update manual or somehow automatic? 
Also It needs to be tested in 33.7, since that's the official supported version.